auth/auth.go

```diff
@@ -0,0 +1,149 @@
+package auth
+
+import (
+	"errors"
+	"net/http"
+	"os"
+	"fmt"
+	"io/ioutil"
+	"crypto/rsa"
+	"time"
+
+	"github.com/DarthHater/bored-board-service/database"
+	"github.com/DarthHater/bored-board-service/model"
+	"github.com/DarthHater/bored-board-service/constants"
+	"github.com/gin-gonic/gin"
+	"github.com/dgrijalva/jwt-go"
+	log "github.com/sirupsen/logrus"
+)
+
+var (
+	signKey   *rsa.PrivateKey
+	verifyKey *rsa.PublicKey
+)
+
+// ReadAndSetKeys will read public and private RSA keys and create a key for signing JWTs.
+func ReadAndSetKeys() {
+	privKeyPath := os.Getenv("PRIVATE_KEY_PATH")
```

Comment:
You might use viper instead of os.Getenv (you could set a default path with viper)